scala/src/main/scala/pekko/contrib/persistence/mongodb/MongoDataModel.scala

```diff
@@ -48,17 +48,27 @@ case class Serialized[C <: AnyRef](bytes: Array[Byte],
 
   val hint = "ser"
   lazy val content: C = {
-
     val clazz = loadClass.getClassFor[X forSome { type X <: AnyRef }](className)
+    def backwardsCompatClazz = loadClass.getClassFor[X forSome { type X <: AnyRef }](className.replace("akka", "org.apache.pekko"))
```

Comment:
`className.replace("akka", "org.apache.pekko")` would replace any occurrences of `akka` ... 
Couldn't we check for `startsWith("akka.")` and if the class starts with "akka." - only then replace the first occurrence using `replaceFirst(regex, "org.apache.pekko")` with `org.apache.pekko`?